app/src/main/res/layout/fragment_search.xml

```diff
@@ -6,13 +6,21 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     tools:context=".mainpage.search.SearchFragment">
 
-
-    <androidx.appcompat.widget.SearchView
+    <SearchView
         android:id="@+id/search_text"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        app:layout_constraintTop_toTopOf="parent"
-        app:queryHint="Search movie" />
+        android:layout_height="35dp"
+        android:layout_margin="15dp"
+        android:layout_marginTop="40dp"
```

Comment:
Pls make use of sdp values here instead of dp 😇

Reply:
Yeah, I have done the changes. Hope it solves the issues.